client/my-sites/plans/plan-overview/plan-status/style.scss

```diff
@@ -72,3 +72,16 @@
 		font-size: 21px;
 	}
 }
+
+.plan-status__nudge {
+	color: $alert-green;
+
+	del {
+		color: initial;
+		font-style: italic;
```

Comment:
We could move this line to `.plan-status__nudge` and then remove the `<em>` component entirely.